examples/kubernetes/topology/immediate-binding/pvc-immediate.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolumeClaim
```

Comment:
Why not put this in the same yaml as the immediate deployment, as you've done with the delayed deployment? I find that it's easier to navigate fewer files, if possible.

Reply:
Done